Packages/vcs/Lib/vtk_ui/text.py

```diff
@@ -87,45 +186,146 @@ def __init__(self, interactor, string, movable=False, on_move=None, on_drag=None
         else:
             self.actor = text_actor(string, fgcolor, size, font)
 
-        widget.SetTextActor(self.actor)
+        widget = WidgetReprShim(interactor, self.actor)
 
         super(Label, self).__init__(interactor, widget)
 
-        #self.widget.ResizableOff()
         self.movable = movable
         self.action = on_click
-        self.release_action = on_release
         self.move_action = on_move
         self.dragged = on_drag
 
-        self.left, self.top = left, top
-        self.top_offset = 0
-
-        # Assigned by Widget.__init__
-        self.repr.MovingOff()
-        self.repr.PickableOff()
-        self.repr.SetShowBorderToOff()
-
         self.actor.SetTextScaleModeToNone()
+        self.actor.SetUseBorderAlign(False)
+        self.actor.VisibilityOff()
 
-        # Map events to draggable actions, because standard events aren't propagated
-        self.add_event_handler("StartInteractionEvent", self.drag_clicked)
-        self.add_event_handler("InteractionEvent", self.drag_moved)
-        self.add_event_handler("EndInteractionEvent", self.drag_released)
-        self.add_event_handler("StartInteractionEvent", self.click)
-        self.add_event_handler("EndInteractionEvent", self.release)
+        self.left = left
+        self.top = top
 
         self.register()
 
-    def get_text(self):
-        return self.repr.GetText()
+    def text():
```

Comment:
I may have gone on a property spree. This one was just because I already had the getter/setter implemented and I needed to access the text value a **lot** in the Textbox.